src/index.js

```diff
@@ -1 +1,27 @@
-import './style.css';
\ No newline at end of file
+import './styles.css';
+import displayScore from './crud';
+import API_URL from './working';
+import postData from './display';
+
+const formAdd = document.querySelector('#form-add');
+const refresh = document.querySelector('#refresh-btn');
+const { name, score } = formAdd.elements;
+
+const addScore = (e) => {
+  e.preventDefault();
+  if (!name.value || !score.value) return;
+  const scoreData = {
+    user: name.value,
+    score: score.value,
+  };
```

Comment:
- [ ] [OPTIONAL] It will be better if you add the function to reset the form inputs after submission 😃 This enhances user experience 😉